source/pkg/reconciler/streamsource/controller.go

```diff
@@ -85,7 +85,17 @@ func NewController(
 			r.updateRedisConfig(ctx, configMap)
 		})
 	} else if !apierrors.IsNotFound(err) {
-		logging.FromContext(ctx).With(zap.Error(err)).Fatal("Error reading ConfigMap'")
+		logging.FromContext(ctx).With(zap.Error(err)).Info("Error reading Redis ConfigMap'")
+	}
+
+	// Get TLS config map and set TLS certificate, to pass data to receive adapter.
+	// Not rolling out new adapters on watch change.
+	if _, err := kubeclient.Get(ctx).CoreV1().ConfigMaps(system.Namespace()).Get(ctx, TLSConfigMapName(), metav1.GetOptions{}); err == nil {
```

Comment:
Usually certificates expire. What's the story for certificate rotation?

Reply:
I think its the same as `config-redis`'s `numConsumers` on-demand update that was requested by asynch team. When a new cert is added to the config-tls CM, we'll need the watch to update the reconciler and create a new redis connection, I assume. 
I can add this to our existing issue for [`config-redis`](https://github.com/knative-sandbox/eventing-redis/issues/8) or create a separate one. That ok? cc: @lionelvillard

Reply:
sounds good!